assets/lib/commands/out.rb

```diff
@@ -66,6 +66,10 @@ def output
         metadata << { 'name' => 'comment', 'value' => comment }
       end
 
+      if params.label
+        Octokit.add_labels_to_an_issue(input.source.repo, id, [params.label])
```

Comment:
Do you know if this appends or replaces all the issues?

Reply:
It appends - there's a separate API call for replace-all.